packages/kit/src/core/sync/create_manifest_data/index.js

```diff
@@ -14,18 +14,34 @@ import { parse_route_id } from '../../../utils/routing.js';
  *   rest: boolean;
  *   type: string | null;
  * }} Part
+ */
+
+/**
+ * @typedef {{
```

Comment:
can we add a comment as to what this is supposed to represent? e.g. "A route along with the page and endpoint corresponding to that route"

Reply:
added something. tbh these types probably won't last long, i'd like to collapse all this a bit. but that'll probably happen as part of layout endpoints